src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  display: flex;
+  position: relative;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  &__logo {
+    position: absolute;
+    width: $sizeicon;
+    height: $sizeicon;
+    left: 50px;
+    top: 10px;
+  }
+
+  &__nav_list {
+    display: flex;
+    position: relative;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__nav_link {
+    display: block;
+    text-transform: uppercase;
+    color: #000;
+    text-decoration: none;
+    padding: 23px 0;
+  }
+
+  &__nav_link:hover {
```

Comment:
To make all the changes smooth on hover, add a transition property for smooth color change on hover. Example: 'transition: color 0.3s'